code/game/objects/items/documents.dm

```diff
@@ -69,18 +74,119 @@
 	w_class = WEIGHT_CLASS_TINY
 	throw_range = 1
 	throw_speed = 1
+	///time required to print a report
+	var/print_time = 5 SECONDS
+	///determines the sound that plays when printing a report
+	var/print_sound_mode = INSPECTOR_PRINT_SOUND_MODE_NORMAL
+	///Power cell used to power the scanner. Paths g
+	var/obj/item/stock_parts/cell/cell = /obj/item/stock_parts/cell/crap
+	///Cell cover status
+	var/cell_cover_open = FALSE
+	///Power used per print in cell units
+	var/power_per_print = INSPECTOR_POWER_USAGE_NORMAL
+	///Power used to say an error message
+	var/power_to_speak = 1
+
+/obj/item/inspector/Initialize()
+	. = ..()
+	if(ispath(cell))
+		cell = new cell(src)
+
+// Clean up the cell on destroy
+/obj/item/clothing/suit/space/Destroy()
+	if(cell)
+		QDEL_NULL(cell)
+	return ..()
+
+// Clean up the cell on destroy
+/obj/item/inspector/handle_atom_del(atom/A)
+	if(A == cell)
+		cell = null
+	return ..()
+
+// support for items that interact with the cell
+/obj/item/inspector/get_cell()
+	return cell
 
 /obj/item/inspector/attack_self(mob/user)
 	. = ..()
-	if(do_after(user, 5 SECONDS, target = user, progress=TRUE))
-		print_report()
+	if(do_after(user, print_time, target = user, progress=TRUE))
+		print_report(user)
 
-///Prints out a report for bounty purposes, and plays a short audio blip.
-/obj/item/inspector/proc/print_report()
-	// Create our report
+/obj/item/inspector/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_CROWBAR)
+		cell_cover_open = !cell_cover_open
+		balloon_alert(user, "You [cell_cover_open ? "open" : "close"] the cell cover on \the [src].")
+	else if(cell_cover_open && istype(I, /obj/item/stock_parts/cell))
```

Comment:
```suggestion

/obj/item/inspector/crowbar_act(mob/living/user, obj/item/crowbar)
	. = ..()
	if(user.combat_mode)
		return
	cell_cover_open = !cell_cover_open
	balloon_alert(user, "You [cell_cover_open ? "open" : "close"] the cell cover on \the [src].")
	return TRUE


/obj/item/inspector/attackby(obj/item/I, mob/user, params)
	if(cell_cover_open && istype(I, /obj/item/stock_parts/cell))
```
* Usage of crowbar_act()
* I guess we'll want some bashing or something like that on combat mode.